src/main/scala/Switcher.scala

```diff
@@ -109,7 +109,7 @@ class TLSwitcher(
         address    = address(i),
         resources  = device.reg("mem"),
         regionType = if (cacheable) RegionType.UNCACHED
-                     else RegionType.UNCACHEABLE,
+                     else RegionType.IDEMPOTENT,
```

Comment:
We might want this to be `VOLATILE`

Reply:
Yeah I feel like the common use for this would be some offchip memory that could be anything so setting it to idempotent seems less safe.